Userland/Libraries/LibDebug/Dwarf/LineProgram.h

```diff
@@ -108,7 +109,7 @@ class LineProgram {
     AK_MAKE_NONMOVABLE(LineProgram);
 
 public:
-    explicit LineProgram(DwarfInfo& dwarf_info, SeekableStream& stream);
+    static ErrorOr<NonnullOwnPtr<LineProgram>> try_create(DwarfInfo& dwarf_info, SeekableStream& stream);
```

Comment:
https://github.com/SerenityOS/serenity/blob/master/Documentation/Patterns.md#fallible-constructors

Please use `create` instead of `try_create`